components/backup-stream/src/metadata/keys.rs

```diff
@@ -129,6 +132,17 @@ impl MetaKey {
         )
     }
 
+    /// defines the key of storage checkpoint-ts of task in a store.
+    pub fn storage_checkpoint_of(name: &str, store_id: u64) -> Self {
```

Comment:
Any tests?

Reply:
This function is called by `set_storage_checkpoint()` and `get_storage_checkpoint()` that is tested by `test_storage_checkpoint()`.

And this key-value written by tikv on etcd and read by br, which is tested with manual case.